src/ssr-template-helper.js

```diff
@@ -54,7 +54,7 @@ export class SsrTemplateHelper {
    * @return {boolean}
    */
   isSupported() {
-    return this.viewer_.canRenderTemplates();
+    return this.viewer_.hasCapability(Capability.VIEWER_RENDER_TEMPLATE);
```

Comment:
The `Capability` enum should either have all of the capabilities listed in CAPABILITIES.md or be removed outright.

Reply:
removed..as much as I'd love to find all the the viewer capability types and create enums for them and replace them throughout the codebase.